sycl/test/optional_kernel_features/atomic_ref-atomic64-aspect.cpp

```diff
@@ -1,52 +0,0 @@
-// RUN: %clangxx %s -S -o %t.ll -fsycl-device-only
-// RUN: FileCheck %s --input-file %t.ll
-
-// CHECK: !sycl_types_that_use_aspects = !{![[#MDNUM1:]], ![[#MDNUM2:]], ![[#MDNUM3:]]}
```

Comment:
This is an integration tests which is supposed to check that our headers lead to generation of right LLVM IR. I don't think that just removing them is a good idea. We should use `-disable-llvm-passes` for this (and similar tests) - this should disable the cleanup and the test shouldn't be dependent on any pass, because it is FE who generates that metadata for us

Reply:
Added -disable-llvm-passes, thanks!